src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_lord_jaraxxus.cpp

```diff
@@ -21,6 +21,7 @@
 #include "SpellScript.h"
 #include "Player.h"
 #include "trial_of_the_crusader.h"
+#include <math.h>
```

Comment:
please use the C++ header name <cmath> instead of the C header <math.h> (as long as it doesn't cause issues I don't know about)